res/graphics/quest icons/fuel_self_service.svg

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think the icon is a bit too busy, but I'm fine with it.

However, could you run it through `svgo` or save it as "Optimized SVG" in Inkscape to clean up unneeded stuff, please?

Reply:
>  I think the icon is a bit too busy, but I'm fine with it.

I'm not a big fan neither, I'm not good at designing things 🙄 By the way I can't find on the web how to mask the icon properly (I noticed that my image has part of the icon outside the circle when viewed in Firefox, but it's ok in the app.

> However, could you run it through svgo or save it as "Optimized SVG" in Inkscape to clean up unneeded stuff, please?

Sure!